Package.swift

```diff
@@ -51,7 +51,7 @@ let package = Package(
         ),
     ],
     dependencies: [
-        .package(url: "https://github.com/tuist/XcodeProj.git", .upToNextMajor(from: "8.0.0")),
+        .package(url: "https://github.com/tuist/XcodeProj.git", .upToNextMajor(from: "8.2.0")),
```

Comment:
There's also a `Project.swift` that you need to update as well. We'd have liked to remove `Package.swift` completely in the future but for now we have to do so manually.

Reply:
Done @fortmarek

Reply:
Sorry, but you'll need to also regenerate the project, so `.package.resolved` gets updated 🙈

Reply:
Done 😛 @fortmarek

Reply:
Thanks!!